content/en/continuous_integration/troubleshooting.md

```diff
@@ -155,11 +155,14 @@ If you have admin access, you can update it from the [Repository Settings Page][
 
 - Your repository needs to have a commit history of at least two commits in the past month.
 - You need to have collected test code coverage in past commits.
```

Comment:
Can we say something here about how this is obtained? E.g. running your tests while Intelligent Test Runner is enabled for the Test Service.

Or whatever it might be.